custom-connector-sdk/src/main/java/com/amazonaws/appflow/custom/connector/model/connectorconfiguration/auth/OAuth2ContentType.java

```diff
@@ -0,0 +1,8 @@
+package com.amazonaws.appflow.custom.connector.model.connectorconfiguration.auth;
+
+public enum OAuth2ContentType {
```

Comment:
Can we add the Java doc for the what this content type mean and how its used and same for the enum types.